components/builder-originsrv/src/data_store.rs

```diff
@@ -1555,33 +1555,6 @@ impl DataStore {
         occ
     }
 
-    pub fn list_origin_channels(
```

Comment:
I am wondering if we should wait to strip things out of originsrv till later... may make it easier to roll back api if needed, and only change one service at a time..